src/note.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  SolidDataset, Thing,
+  getThing, createThing,
+  addUrl,
+  setInteger, setDecimal, setStringNoLocale, setBoolean,
+  getStringNoLocale, getInteger, getBoolean, getUrl,
+} from '@inrupt/solid-client';
+import { createNS } from "@ontologies/core"
+
+import {arrayToThings, thingsToArray} from "./collections"
+
+const noteNSUrl = "https://mysilio.garden/ontologies/note#"
+const noteNS = createNS(noteNSUrl)
```

Comment:
I use `@rdf-esm/namespace` for this:
```import namespace from '@rdf-esm/namespace```

See `src/vocab` for example usage.

Reply:
Doesn't matter much which one we use, I don't think, because they both build NamedNodes. I think I even came across the @ontologies one, might be by the same guy?